frontend/src/component/common/TooltipLink/TooltipLink.tsx

```diff
@@ -6,32 +6,49 @@ import {
 } from '../HtmlTooltip/HtmlTooltip';
 
 const StyledLink = styled(Link, {
-    shouldForwardProp: (prop) => prop !== 'highlighted',
-})<{ highlighted?: boolean }>(({ theme, highlighted }) => ({
-    backgroundColor: highlighted ? theme.palette.highlight : 'transparent',
-    color: theme.palette.text.primary,
-    textDecorationColor: theme.palette.text.disabled,
-    textDecorationStyle: 'dashed',
-    textUnderlineOffset: theme.spacing(0.5),
-    whiteSpace: 'nowrap',
-}));
+    shouldForwardProp: (prop) => prop !== 'highlighted' && prop !== 'clampText',
+})<{ highlighted?: boolean; clampText?: boolean }>(
+    ({ theme, highlighted, clampText }) => ({
+        backgroundColor: highlighted ? theme.palette.highlight : 'transparent',
+        color: theme.palette.text.primary,
+        textDecorationColor: theme.palette.text.disabled,
+        textDecorationStyle: 'dashed',
+        textUnderlineOffset: theme.spacing(0.5),
+        whiteSpace: 'nowrap',
+        ...(clampText
```

Comment:
When I was clamp I though we gonna use css clamp so maybe we should pick a term that doesn't have other meaning in CSS?

Reply:
`overflowEllipsis`?